tests/test_strategy_infinite_bus.py

```diff
@@ -15,22 +15,35 @@
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
-import pytest
+import os
 import sys
-import pendulum
 from uuid import uuid4
 
-from d3a.models.market.market_structures import Offer, Trade, BalancingOffer, Bid
-from d3a.models.strategy.infinite_bus import InfiniteBusStrategy
-from d3a.models.area import DEFAULT_CONFIG
-from d3a.d3a_core.device_registry import DeviceRegistry
+import pendulum
+import pytest
 from d3a_interface.constants_limits import ConstSettings, GlobalConfig
-from d3a.constants import TIME_ZONE
 from d3a_interface.utils import find_object_of_same_weekday_and_time
 
+from d3a.constants import TIME_ZONE
+from d3a.d3a_core.device_registry import DeviceRegistry
+from d3a.d3a_core.util import d3a_path
+from d3a.models.area import DEFAULT_CONFIG
+from d3a.models.market.market_structures import Offer, Trade, BalancingOffer, Bid
+from d3a.models.strategy.infinite_bus import InfiniteBusStrategy
+
 TIME = pendulum.today(tz=TIME_ZONE).at(hour=10, minute=45, second=0)
 
 
+@pytest.fixture(scope="function", autouse=True)
+def auto_fixture():
+    ConstSettings.IAASettings.MARKET_TYPE = 1
+    ConstSettings.BalancingSettings.ENABLE_BALANCING_MARKET = True
+    GlobalConfig.sim_duration = pendulum.duration(days=1)
+    GlobalConfig.slot_length = pendulum.duration(minutes=15)
```

Comment:
Just to double-check: will these two constants be reverted to their default value once the tests are done?

Reply:
Deleted these calls completely because I found out the test that did not revert these variables in the first place, therefore these should not be needed anymore.